parse-args.js

```diff
@@ -62,6 +62,14 @@ function parseArgs () {
     type: 'string',
     describe: 'execute string as if inside `npm run-script`'
   })
+  .option('shell-auto-fallback', {
+    choices: ['', 'bash', 'fish', 'zsh'],
```

Comment:
I'll re-test, but I think yargs refused to parse the option if the default wasn't in the choices. Although. Given the difficulty of figuring out the shell from a subprocess, maybe I should just remove both the autodetecting code and the default value here, and just do it based on the passed value.